Releasenotes.md

```diff
@@ -0,0 +1,77 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-historie-bevragen API als release is uitgebracht zijn de specificaties van deze endpoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar redelijk stabiel geweest.
+Voor het uitbrengen van release v1.0.0 is besloten om de historie-endpoints in een aparte API onder te brengen en daarvoor een eigen github-repository te maken. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Dit is een globaal (en niet limitatief) overzicht van wijzigingen die doorgevoerd zijn voordat v1.0.0 is uitgebracht. Enkele van deze wijzigingen zijn breaking ten opzichte van v0.9.0.
+
+We bieden nu ook SDK's aan met gegenereerde [plumbing-code](./code). Daarnaast bieden we een[postman-collectie](./test) t.b.v. testen aan.
```

Comment:
spatie voor [postman-collectie]

Reply:
Deze zin is bij een latere wijziging weggevallen. Ik voeg hem weer toe, inclusief spatie voor de [postman-collectie]